sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/FeedbackModel.kt

```diff
@@ -3,6 +3,7 @@ package co.touchlab.sessionize
 import co.touchlab.sessionize.api.FeedbackApi
 import co.touchlab.sessionize.api.NetworkRepo
 import co.touchlab.sessionize.db.SessionizeDbHelper
+import co.touchlab.sessionize.platform.NotificationsModel.cancelFeedbackNotifications
 import co.touchlab.sessionize.jsondata.Session
 import co.touchlab.sessionize.platform.Date
 import co.touchlab.sessionize.platform.NotificationsModel.cancelFeedbackNotificationsForSession
```

Comment:
Sorry, this is actually the line that's causing the compile problem and was removed from NotificationsModel. I'm going to mark the last comment I made as resolved.